helpers/make_network_dirs_main.py

```diff
@@ -0,0 +1,118 @@
+
+    for year in years:
+        for survey in surveys:
+
+            tree = {
+                f"{year}_surveys": {
+                    f"{survey}": {
+                        "01_staging": {
+                            "feather": {},
+                            "staging_qa": {
+                                "full_responses_qa": {},
+                                "postcode_validation": {},
+                                "Postcodes for QA": {},
+                            },
+                        },
+                        "02_freezing": {
+                            "changes_to_review": {},
+                            "freezing_updates": {},
+                            "frozen_data_staged": {},
+                        },
+                        "03_northern_ireland": {
+                            f"{year}": {},
+                            "ni_staging_qa": {},
+                        },
+                        "04_construction": {
+                            "manual_construction": {},
+                        },
+                        "05_mapping": {
+                            "mapping_qa": {},
+                        },
+                        "06_imputation": {
+                            "backdata_output": {},
+                            "imputation_qa": {},
+                            "manual_trimming": {},
+                        },
+                        "07_outliers": {
+                            "auto_outliers": {},
+                            "manual_outliers": {},
+                            "outliers_qa": {},
+                        },
+                        "08_estimation": {
+                            "estimation_qa": {},
+                        },
+                        "09_apportionment": {
+                            "apportionment_qa": {},
+                        },
+                        "10_outputs": {
+                            "output_frozen_group": {},
+                            "output_fte_total_qa": {},
+                            "output_gb_sas": {},
+                            "output_intram_by_civil_defence": {},
+                            "output_intram_by_pg_gb": {},
+                            "output_intram_by_pg_uk": {},
+                            "output_intram_by_sic": {},
+                            "output_intram_gb_itl1": {},
+                            "output_intram_gb_itl2": {},
+                            "output_intram_totals": {},
+                            "output_intram_uk_itl1": {},
+                            "output_intram_uk_itl2": {},
+                            "output_long_form": {},
+                            "output_ni_sas": {},
+                            "output_short_form": {},
+                            "output_status_filtered_qa": {},
+                            "output_tau": {},
+                        },
+                    },
+                    "mappers": {
+                        "v1": {},
```

Comment:
the mappers dictionary can also be outside the survey_type loop, it is just created once per year.